### DISCUSSION TITLE
Transient v0.3.7 released

### DISCUSSION THREAD

**Discussion by tarsius** [magit/transient#171] [Announcements]
Please see the [Release Notes](https://github.com/magit/transient/blob/master/docs/CHANGELOG)

<hr /><em>This discussion was created from the release <a href='https://github.com/magit/transient/releases/tag/v0.3.7'>v0.3.7</a>.</em>

**Comment by dave-f** [comment]
Not sure if this is related to the latest version, but I have a simple transient to run a command line app which is really useful:

```
(transient-define-prefix fireevents-transient ()
       "A transient for running events"
       :value '("--verbose" "--events=1" "--port=123" "--read=1" "--log=C:/Dave/events.log")
       ["Logging"
        ("-l" "Log to named file (default events.log)" "--log=")
        ("-v" "Use verbose logging" "--verbose")]
       ["Actions"
        ("f" "Fire events" run-fireevents)])
```

And now if I press eg "-l" I get:

`Unbound slot: transient-prefix, "#<transient-prefix transient-prefix-89df80>", incompatible, oref`

**Reply by tarsius** [reply]
Have you tried restarting Emacs? I cannot reproduce and don't see how that could happen, this slot is initialized to nil, not left unbound.

Also the arguments/values in :value that do not have a corresponding entry in "Logging" are silently dropped. That was always the case.

**Reply by dave-f** [reply]
Ah yes simply restarting Emacs sorted it - very sorry to have wasted your time, thanks for the quick response!
